src/lib/common/tag.h

```diff
@@ -130,7 +128,6 @@ extern std::string valueTag
   const std::string&  value,
   Format              format,
   bool                showComma           = false,
-  bool                isAssociation       = false,
   bool                valueIsNumberOrBool = false
 );
 
```

Comment:
These are the 3 valueTag() functions defined in tag.h:

```
extern std::string valueTag
(
  const std::string&  indent,
  const std::string&  tagName,
  const std::string&  value,
  Format              format,
  bool                showComma           = false,
  bool                isVectorElement     = false
);

extern std::string valueTag
(
  const std::string&  indent,
  const std::string&  tagName,
  int                 value,
  Format              format,
  bool                showComma     = false
);

extern std::string valueTag
(
  const std::string&  indent,
  const std::string&  xmlTag,
  const std::string&  jsonTag,
  const std::string&  value,
  Format              format,
  bool                showComma           = false,
  bool                valueIsNumberOrBool = false
);
```

As far as I understand, where would be the conflict? It seems that all the signatures are differents (including the "variability" that the usage or not usage of optional arguments may introduce).